### ISSUE TITLE
RFC suggestion/discussion

### ISSUE THREAD

**Issue by brendt** [brendt/rfc-vote#293]
Potential idea, I like it a lot: https://x.com/ryangjchandler/status/1700564448426029399?s=46

**Comment by SerhiiCho** [comment]
What about GitHub discussions? Maybe they will be useful?

**Comment by SerhiiCho** [comment]
In case of Laravel they have a great [discussions page](https://github.com/laravel/framework/discussions) on github

**Comment by gaborjonas** [comment]
I was thinking about exactly the same thing, happy to contribute if needed.

**Comment by SerhiiCho** [comment]
@gaborjonas it's not the same thing

**Comment by SerhiiCho** [comment]
@gaborjonas, yeah, you changed my mind. It does make sense to make it on rfc.stitcher.io
